common/src/main/java/com/wynntils/features/inventory/HightlightDuplicateCosmeticsFeature.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.inventory;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.Config;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.mc.event.ContainerRenderEvent;
+import com.wynntils.mc.event.ContainerSetContentEvent;
+import com.wynntils.mc.event.ScreenClosedEvent;
+import com.wynntils.mc.event.ScreenInitEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.models.containers.type.SearchableContainerType;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.render.RenderUtils;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.inventory.ChestMenu;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.INVENTORY)
+public class HightlightDuplicateCosmeticsFeature extends Feature {
+    @Persisted
+    public final Config<HighlightCondition> highlightCondition = new Config<>(HighlightCondition.BOTH);
+
+    @Persisted
+    public final Config<CustomColor> hoveredHighlightColor = new Config<>(CommonColors.BLUE);
+
+    @Persisted
+    public final Config<CustomColor> selectedHighlightColor = new Config<>(CommonColors.ORANGE);
+
+    private static final Component ADD_REWARD_TEXT = Component.literal("§7Click on a reward to add it");
+    // The colored glass panes on the new reward screen
+    private static final Component DECORATIVE_ITEM_NAMES = Component.literal("§f");
+    private static final Component RETURN_TEXT = Component.literal("§7Return to Scrap Menu");
+    private static final int FINAL_SLOT = 53;
+    private static final int RETURN_SLOT = 18;
+    private static final List<Integer> SELECTED_COSMETIC_SLOTS = List.of(1, 2, 3, 4, 5);
```

Comment:
Might be too much info for a feature. Do you want to move this logic into a model?

Reply:
Although, moving only these constants would be weird in code. Since the events/logic is appropriate here. Okay, perhaps keep this for now, and if we ever extend it, move to a model. Unless you have a better idea.

Reply:
I think there is some kind of minimum threshold for creating a new model. If we have an existing model, we should prefer to enhance it rather than putting too much logic in a feature, but in absence of an existing model, I think we can accept some amount of "model-y" logic in a feature. This is of course a subjective judgement, but agree that it seems reasonable to keep this in the feature for now.

Reply:
Potentially if this type of feature is used elsewhere, like in the TM as DonkeyBlaster suggested, then it could be moved to a Model but for now yeah it's fine here I think